internal/ctl/approve.go

```diff
@@ -0,0 +1,25 @@
+package ctl
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// approveCmd represents the approve command
+var approveCmd = &cobra.Command{
+	Use:   "approve",
+	Short: "Approve is used to enable a telephone profile connection",
```

Comment:
Bit of word soup here.  The profiles belong to locksmith, the connections belong to wireguard and are driven by locksmith.  As the language is generally inconsistent throughout this PR, I'm only flagging this one line for review, but all occurrences should be checked.